govcd/vm_test.go

```diff
@@ -991,11 +991,14 @@ func (vcd *TestVCD) Test_AddInternalDisk(check *C) {
 	//cleanup
 	err = vm.DeleteInternalDisk(diskId)
 	check.Assert(err, IsNil)
+
+	// disable fast provisioning if needed
+	updateVdcFastProvisioning(vcd, check, previousProvisioningValue)
```

Comment:
Could there be a cleanup task (using AddToCleanupList) so that if the test crashes we still have the ability to restore the environment to what it was.

Reply:
added